src/mixins/portfolio.js

```diff
@@ -305,6 +305,27 @@ export const createPortfolioMixin = ({
     currentNFTClassSortingOptionList() {
       const options = [];
 
+      const currentClassList =
+        this.currentTab === tabOptions.collected
+          ? this.nftClassListOfCollected
+          : this.nftClassListOfCreated;
+
+      if (
+        currentClassList.some(({ classId }) =>
+          this.nftClassMapOfNFTBook.has(classId)
+        ) ||
+        currentClassList.some(({ classId }) =>
+          this.nftClassMapOfOther.has(classId)
+        )
+      ) {
+        this.nftClassListOfCollectedSorting = NFT_CLASS_LIST_SORTING.TYPE;
+        this.nftClassListOfCreatedSorting = NFT_CLASS_LIST_SORTING.TYPE;
```

Comment:
We should never have sideeffect in `computed` properties, they should be pure functional
i.e. no `this.xxx = yyy` inside any `computed` functions.